frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/load_mesh_menu_item.tsx

```diff
@@ -0,0 +1,34 @@
+import { WarningOutlined } from "@ant-design/icons";
+import FastTooltip from "components/fast_tooltip";
+import type { VolumeTracing } from "oxalis/store";
+import type { APIMeshFile } from "types/api_flow_types";
+
+type Props = {
+  currentMeshFile: APIMeshFile | null | undefined;
+  volumeTracing: VolumeTracing | null | undefined;
+};
+
+export function LoadMeshMenuItemLabel({ currentMeshFile, volumeTracing }: Props) {
```

Comment:
The name does not match with the file name. Maybe the file name should be adjusted a little